package.json

```diff
@@ -64,8 +64,8 @@
     "lodash.sortby": "4.7.0",
     "match-sorter": "6.3.1",
     "react-autosuggest": "10.1.0",
-    "react-modal": "3.14.4",
-    "react-toastify": "8.2.0",
+    "react-modal": "3.15.1",
+    "react-toastify": "9.0.5",
```

Comment:
Major version upgrade that added some new features. The breaking changes don't affect us and notifcations are still working just fine in the storybook and in app.